print("Dentro do próprio módulo pode se usar blocos try...except para capturar e lidar com exceções específicas que possam ocorrer durante a execução do código. Se uma exceção não puder ser tratada localmente, você pode optar por propagá-la para o código que chamou o módulo. Gerar arquivos de log.")
